src/inject/GPMInject/interface/mini.js

```diff
@@ -16,7 +16,7 @@ window.wait(() => {
     Emitter.fire('mini', { state: true });
     oldSize = remote.getCurrentWindow().getSize();
     mainWindow.setSize(MINI_SIZE, MINI_SIZE);
-    mainWindow.setMaximumSize(MINI_SIZE, MINI_SIZE);
```

Comment:
Perhaps .setAspectRatio(1) could be used? Restricting the maximum size doesn't prevent the issue of resizing smaller at say 200x250, and I would like it to be flexible enough to work with different people's workflows (namely, mine :-p).

IMO, the album art may be a 300x300 image, but it is not guaranteed to be "crisp and clear" at that resolution. Source: tested resizing larger and smaller and image quality doesn't seem to change when it's not it's native 300^2.